httpapi/coreapi/types.go

```diff
@@ -78,9 +92,19 @@ type DocumentResponse struct {
 func toDocumentAttributes(cattrs map[string]AttributeRequest) (map[documents.AttrKey]documents.Attribute, error) {
 	attrs := make(map[documents.AttrKey]documents.Attribute)
 	for k, v := range cattrs {
-		attr, err := documents.NewAttribute(k, documents.AttributeType(v.Type), v.Value)
-		if err != nil {
-			return nil, err
+		var attr documents.Attribute
+		var err error
+		switch documents.AttributeType(v.Type) {
+		case documents.AttrMonetary:
+			attr, err = documents.NewMonetaryAttribute(k, v.MonetaryValue.Value, v.MonetaryValue.ChainID.Bytes(), []byte(v.MonetaryValue.ID))
```

Comment:
this will panic here if I set the type to monetery but dont give any data. Lets change the v.MoneetaryValue to value instead of pointer?